### PR TITLE
uniq: add page

### PR DESCRIPTION
**Pull request by muteness**
added a new page for **uniq** command.

See discussion in #653.

### CODE REVIEW

**Comment by igorshubovych** [comment]
This is related to #653

**Comment by muteness** [comment]
That PR is closed because of some git issues. I opened this new PR instead.

**Comment by igorshubovych** [comment]
@muteness yes, i know. I just let other reviewers to know.

File: pages/common/uniq.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# uniq
+
+> Report or omit repeated lines.
+
+- Display each line once.
+
+`uniq {{file}}`
+
+- Display occurence of each line along with that line.
```
Comment (rubenvereecken) [inline pages/common/uniq.md]: occurrence

**Comment by rubenvereecken** [comment]
Just got the one inline comment. That and we recently switched to a different syntax for example descriptions. Really sentences just end in colons instead of periods. Have a look at [any random page](https://github.com/tldr-pages/tldr/blob/master/pages/common/ack.md)

File: pages/common/uniq.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/uniq.md]: :+1: @rubenvereecken, I was thinking the same thing.

**Comment by muteness** [comment]
Just fixed the two issues.

**Comment by igorshubovych** [comment]
LGreatTM

**Comment by waldyrious** [comment]
Doesn't `-c` imply `-u`? Maybe it should appear after it.

**Comment by muteness** [comment]
I don't quite understand your logic here.

And I think they are used for different purposes.

**Comment by rubenvereecken** [comment]
Alright I'll go with @waldyrious on this one: the very first use case for `uniq` would be to `uniqueify` a file or other, then for counting occurrences. It'd be great if you could switch 1st and 3rd example.

**Comment by muteness** [comment]
**uniq {{file}} ** displays each line once which means if one line repeatedly appears in a file, the output just shows that line once.

**Comment by waldyrious** [comment]
Oh yeah, I was mixing it up with uniq -u, nevermind. LGTM, thanks for the contribution!

**Pull request merged**